spec/factories/inning.rb

```diff
@@ -2,5 +2,19 @@
   factory :inning do
     name { '2107' }
     current { true }
+    start_date {Date.today+2.weeks}
+    
+    trait :not_current_past do
+      current { false }
+      start_date {Date.today-5.weeks}
+    end
+    trait :is_current do
+      current { true }
+      start_date {Date.today-2.weeks}
+    end
+    trait :not_current_future do
+      current { false }
+      start_date {Date.today+2.weeks}
+    end
```

Comment:
Love this!

Reply:
We thought this was pretty neat.  Definitely want to explore more of what factory bot has to offer!